packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -85,8 +83,7 @@ export class OdspSummaryUploadManager {
     ): Promise<IWriteSummaryResponse> {
         const { snapshotTree, blobs } = await this.convertSummaryToSnapshotTree(
             parentHandle,
-            // Clone as we change the blob contents.
-            cloneDeep(tree),
```

Comment:
We are not changing the contents anymore due to removal of blob deduping, so removed cloning.